pytest_catchlog.py

```diff
@@ -311,3 +312,17 @@ def caplog(request):
     return CompatLogCaptureFixture(request.node)
 
 capturelog = caplog
+
+
+# A fix for py.builtin._totext
+# see: https://bitbucket.org/pytest-dev/py/issues/75/pybuiltin_totext-does-not-work-with-python
+if sys.version_info >= (3, 0):
+    _totext = py.builtin._totext
+else:
+    def _totext(obj, encoding='utf-8', errors=None):
+        if isinstance(obj, str):
+            if errors is None:
+                obj = obj.decode(encoding)
+            else:
+                obj = obj.decode(encoding, errors)
+        return obj
```

Comment:
I added it here to be able to access it from `testdir.makepyfile`. If there is a batter way how to make `_totext` visible from `testdir.makepyfile`, I can fix it.